src/java.base/share/man/java.md

```diff
@@ -3757,9 +3757,10 @@ general form:
     be loaded on top of those in the `<static_archive>`.
 -   On Windows, the above path delimiter `:` should be replaced with `;`
 
-(The names "static" and "dynamic" are used for historical reasons.
-The only significance is that the "static" archive is loaded first and
-the "dynamic" archive is loaded second).
+The names "static" and "dynamic" are used for historical reasons. The dynamic
+archive, while still useful, supports fewer optimizations than
+available for the static CDS archive. If the full set of CDS/AOT
+optimizations are desired, consider using the AOTCache features describe below.
```

Comment:
```suggestion
optimizations are desired, consider using the AOT cache features described below.
```

Reply:
The word "features" sounds awkward here. I changed "AOTCache features" -> "AOT cache".